hack/check_bindata_conflicts.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env -S uv run --with pyyaml --with requests
```

Comment:
With https://github.com/openshift/kueue-operator/pull/863/files#r2492538226, you can simply make the following change:
```suggestion
#!/usr/bin/env python3
```